test/lib/next-test-utils.js

```diff
@@ -176,11 +180,16 @@ export function runNextCommandDev(argv, stdOut, opts = {}) {
     ...opts.env,
   }
 
+  const hookArgs = opts.hook ? ['-r', opts.hook] : []
   return new Promise((resolve, reject) => {
-    const instance = spawn('node', ['--no-deprecation', nextBin, ...argv], {
-      cwd,
-      env,
-    })
+    const instance = spawn(
+      'node',
+      [...hookArgs, '--no-deprecation', nextBin, ...argv],
```

Comment:
This should be renamed as well right?

Reply:
Add a suspense case to let test failed when react 17 is resolved. I was curious why they passed with these typos...